helm/templates/log-cleanup-cronjob.yaml

```diff
@@ -0,0 +1,55 @@
+{{- if .Values.cleanup.enabled }}
+{{- $_ := set . "notifyBCInstanceType" "log-cleanup" -}}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ include "NotifyBC.fullname" . }}-{{ .notifyBCInstanceType }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "NotifyBC.labels" . | nindent 4 }}
+spec:
+  schedule: "0 3 * * *"
```

Comment:
This runs every day at 3AM which is fine but it doesn't match the PR title ("Add cronjob to clean up logs on the first of every month"). I think it's ok to run daily just wanted to check the intention.

Reply:
Whoops yeah, forgot to change it back after testing. Thanks for the catch.